pkg/git/langs.go

```diff
@@ -1,40 +1,44 @@
 package git
 
 import (
-	"fmt"
 	"sort"
 )
 
 // SortLanguagesWithInts sorts languages from the given map to a slice where the first one is with the highest number
 func SortLanguagesWithInts(langsWithSizes map[string]int) []string {
-	var contentSizes []string
-	reversedMap := map[string]string{}
+	var contentSizes []int
+	reversedMap := make(map[int][]string)
 	for lang, size := range langsWithSizes {
-		key := fmt.Sprintf("%d_%s", size, lang)
-		reversedMap[key] = lang
-		contentSizes = append(contentSizes, key)
+		if _, ok := reversedMap[size]; !ok {
+			contentSizes = append(contentSizes, size)
+		}
+		reversedMap[size] = append(reversedMap[size], lang)
 	}
-	return sortLanguages(contentSizes, reversedMap)
+	sort.Ints(contentSizes)
+
+	var sortedLangs []string
+	for i := len(contentSizes) - 1; i >= 0; i-- {
+		sortedLangs = append(sortedLangs, reversedMap[contentSizes[i]]...)
+	}
+	return sortedLangs
 }
```

Comment:
Have you considered using the [sort.Sort](https://golang.org/pkg/sort/) interface? Maybe you could declare a custom type to sort the languages in reverse order of their size? (just an idea, though)

Reply:
good idea. do you mean something like this: 5eb83c2 ?

Reply:
yes, that looks simpler with one less loop in the code ;)